quickwit/Cargo.toml

```
@@ -106,6 +106,7 @@ opentelemetry-jaeger = { version = "0.17", features = ["rt-tokio"] }
 opentelemetry-otlp = "0.11.0"
 pin-project = "1.0.12"
 pnet = { version = "0.31.0", features = ["std"] }
+postcard = { version = "1.0.4", features = ["alloc"] }
```

I would have expected:
```suggestion
postcard = { version = "1.0.4", features = ["use-std"], default-features = false}
```

---

the alloc stuff is without heapless